src/main/webapp/app/course/manage/course-update.component.ts

```diff
@@ -296,12 +296,8 @@ export class CourseUpdateComponent implements OnInit {
             course['courseInformationSharingConfiguration'] = CourseInformationSharingConfiguration.DISABLED;
         }
 
-        // TODO: this has to be removed once the refactoring from course 'registration' to 'enrollment' is complete
-        course['enrollmentEnabled'] = course['registrationEnabled'];
-        delete course['registrationEnabled'];
-        if (course['enrollmentEnabled'] == true) {
-            course['enrollmentConfirmationMessage'] = course['registrationConfirmationMessage'];
-            delete course['registrationConfirmationMessage'];
+        if (!course['enrollmentEnabled']) {
+            course['enrollmentConfirmationMessage'] = undefined;
```

Comment:
Is there any reason to not use this?
```suggestion
        if (!course.enrollmentEnabled) {
            course.enrollmentConfirmationMessage = undefined;
```